auth_service/user_management/user_registry/models/dto.py

```diff
@@ -90,6 +83,17 @@ class UserCreatableData(BaseDto):
     )
 
 
+class UserRegisteredData(UserBasicData):
+    """Basic data of a registered user"""
+
+    ls_id: EmailStr = Field(
```

Comment:
Could we maybe call it external_id instead of ls_id? This would make it easier if we later have to switch to a different identity provider.

Reply:
Changed in 36ad31a7a1e08e01f8669644f91f918c9d62f4e7 now.